fastlane/Fastfile

```diff
@@ -13,7 +13,6 @@
 files_with_version_number = {
     './.version' => ['{x}'],
     './PurchasesHybridCommon.podspec' => ['s.version          = "{x}"'],
-    './android/build.gradle' => ['versionName "{x}"'],
```

Comment:
versionCode/versionName is not needed in libraries anymore so I took the opportunity to remove it from there